DepositRegistrar.sol

```diff
@@ -0,0 +1,306 @@
+// Name registrar using deposits
+
+contract Deed {
+    // The Deed is a contract intended simply to hold ether
+    // It can be controlled only by the registrar and can only send ether back to the owner
+    address public owner;
```

Comment:
During the bid auction separating both allows greater anonymity. But maybe bidder is unnecessary to keep.